README.md

```diff
@@ -11,75 +11,184 @@ Generative AI is fallible in ways similar to people. It makes typos, misintepret
     - Be transparent: in the account's bio, disclose that the account is a bot
-1. Generate a Personal Access Token[^1]:
+
+2. **Generate Personal Access Token**[^1]:
     - Go to Settings → Developer settings → Personal access tokens → Tokens (classic)
     - Click "Generate new token"
     - Select scopes:
       - `repo` (Full control of private repositories)
       - `workflow` (If the bot should be allowed to modify `.github/workflows`)
-1. Add the bot to a project as a collaborator
-    - Switch to your main GitHub account
-    - Navigate to a GitHub repository that you are the owner of
+
+3. **Add Bot as Collaborator**:
+    - Navigate to your GitHub repository
     - Go to Settings → Collaborators
     - Click "Add people"
-    - Search for your bot account via whatever method you prefer and select it
-    - Click "Add to repository"
+    - Search for your bot account and add it to the repository
     - Switch to your bot account to accept the invite
 
-[^1]: There is currently no way to generate fine-grained access tokens for collaborator access to repositories owned by
-individuals. When you give a classic Personal Access Token to the bot, you should assume from that point on that it can
-and will go rogue and attempt to abuse the broad permissions of that access token. As a repository owner, use
-collaborator permission settings and protected branches to restrict the bot's permissions to only the minimum required
-to perform its intended functions.
-
-### Anthropic API Setup
+4. **Get Anthropic API Key**:
+    - Sign up for an Anthropic account at https://console.anthropic.com
+    - Generate an API key from the console
+    - Ensure you have sufficient credits for API usage
+
+[^1]: There is currently no way to generate fine-grained access tokens for collaborator access to repositories owned by individuals. When you give a classic Personal Access Token to the bot, you should assume from that point on that it can and will go rogue and attempt to abuse the broad permissions of that access token. As a repository owner, use collaborator permission settings and protected branches to restrict the bot's permissions to only the minimum required to perform its intended functions.
+
+#### Setup Instructions
+
+1. **Configure Repository Variables**:
+   - Go to your repository → Settings → Secrets and variables → Actions → Variables tab
+   - Add the following repository variables:
+     - `BOT_USERNAME`: Your bot's GitHub username
+     - `AUTHORIZED_USERNAME`: Your main GitHub username (who can trigger the bot)
+
+2. **Configure Repository Secrets**:
+   - Go to your repository → Settings → Secrets and variables → Actions → Secrets tab
+   - Add the following repository secrets:
+     - `BOT_GITHUB_TOKEN`: The Personal Access Token from your bot account
+     - `ANTHROPIC_API_KEY`: Your Anthropic API key
+
+3. **Add the GitHub Action Workflow**:
+   - Create `.github/workflows/bot.yml` in your repository with the following content:
```

Comment:
We could link to the workflow file within this repository, use that as the source of truth, what do you think?